### ISSUE TITLE
Updater app doesn't go through with Firefox (v.43), but does with Chrome (oc 8.0x to 8.1.5)

### ISSUE THREAD

**Issue by Guillaume00** [owncloud/core#21604]
I've upgraded owncloud 8.0.x on 3 different servers and it has never worked using Firefox (v. 43). When using FF, I would always remain stuck on the maintenance page for hours.

After some research, I found on an old forum thread how to solve my issue:
- manually edit the config.php file to change maintenance mode from true to false
- restart apach
- then launch the upgrade using Chrome

Original thread: https://forum.owncloud.org/viewtopic.php?f=23&t=9024

I've encountered this problem all 3 times, all running ubuntu 14.04 LTS and updating to ownCloud 8.1.5
Every time, Firefox (v. 43) did not work but Chrome did.

**Comment by RealRancor** [comment]
@Guillaume00 Are you using the repositories from download.owncloud.org? If yes the kept maintenance mode is the expected behavior documented at various places.

**Comment by rullzer** [comment]
@Guillaume00 do you observe the same issue if you do the initial upgrade via chrome?

**Comment by Guillaume00** [comment]
@RealRancor I'm using the official repositories mentioned in the manual at owncloud.org

The issue was not that it was kept in maintenance mode. It is that it remained stuck on the first page the updater shows. The one that says it will refresh by itself. And never upgraded anything.
The first page never refreshed. And when I tried manually after waiting more than 2 hours, it showed me the page warning about maintenance going on.

After I got out of maintenance mode and launched the upgrade using Chrome, it refreshed almost immediately, showed some info about progress (if I recall) and got out of maintenance mode all by itself. All of that in barely a few minutes (or less).

@rullzer On the third server, I launched the upgrade using Chrome and it all went smoothly...
